ui/bake.py

```diff
@@ -320,6 +320,9 @@ def draw(self, context):
                     row = col.row(align=True)
                     row.separator()
                     row.prop(context.scene, 'bake_unwrap_angle', expand=True)
+                if context.scene.bake_uv_overlap_correction == "MANUALNOPACK" and any(plat.use_decimation for plat in context.scene.bake_platforms) and (not context.scene.bake_emit_indirect) and (not context.scene.bake_prioritize_face) and (not context.scene.bake_pass_ao):
+                    col.label(text="***Skipping the packing step will not work if one or more of your platforms has decimation active,***") #TRANSLATE PLEASE!
```

Comment:
The emphasis is waaay overdone here. Just add a note that says:
"Warning: may have unexpected results if you have overlapping islands."

Reply:
otherwise looking great! I meant to add something like this forever ago